dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/BasicConfigTests.java

```diff
@@ -126,11 +126,13 @@ public static void setUp() throws Exception {
                                                                                 + ".war/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSource"),
                                                                        "services/org.eclipse.microprofile.config.spi.ConfigSource");
 
-        ShrinkHelper.exportDropinAppToServer(server, customSourcesWar);
-        ShrinkHelper.exportDropinAppToServer(server, types_war);
-        ShrinkHelper.exportDropinAppToServer(server, cdiConfigWar);
-        ShrinkHelper.exportDropinAppToServer(server, convertersWar);
-        ShrinkHelper.defaultDropinApp(server, CONVERTER_PRIORITY_APP_NAME, "com.ibm.ws.microprofile.config11.converter.*");
+        ShrinkHelper.exportDropinAppToServer(server, customSourcesWar, DeployOptions.SERVER_ONLY);
+        ShrinkHelper.exportDropinAppToServer(server, types_war, DeployOptions.SERVER_ONLY);
+        ShrinkHelper.exportDropinAppToServer(server, cdiConfigWar, DeployOptions.SERVER_ONLY);
+        ShrinkHelper.exportDropinAppToServer(server, convertersWar, DeployOptions.SERVER_ONLY);
+        DeployOptions[] options = new DeployOptions[1];
+        options[0] = DeployOptions.SERVER_ONLY;
+        ShrinkHelper.defaultDropinApp(server, CONVERTER_PRIORITY_APP_NAME, options, "com.ibm.ws.microprofile.config11.converter.*");
```

Comment:
I think you can do `DeployOptions[] options = {DeployOptions.SERVER_ONLY};` to reduce the cruft here.

Reply:
It might actually be cleaner to call `buildDefaultApp` and then `exportDropinAppToServer` rather than adding this new method which requires you to construct an array to call it.

Reply:
Actually I was thinking that DeployOptions should be a binary mask rather than an enum ... but that was a much bigger change.

Reply:
Since my PB had a failure, I will look at improving this a little as well

Reply:
Done